lib/solargraph/source_map.rb

```diff
@@ -114,10 +114,13 @@ def locate_named_path_pin line, character
     # @param line [Integer]
     # @param character [Integer]
     # @return [Pin::Namespace,Pin::Method,Pin::Block]
-    def locate_block_pin line, character
-      _locate_pin line, character, Pin::Namespace, Pin::Method, Pin::Block
+    def locate_closure_pin line, character
+      _locate_pin line, character, Pin::Closure
     end
 
+    # @deprecated Please use locate_closure_pin instead
+    alias locate_block_pin locate_closure_pin
```

Comment:
@lekemula Heads up on this one - once this and #877 are out, maybe you can make a new release that uses locate_closure_pin so we can remove this alias.